src/index.js

```diff
@@ -28,6 +29,18 @@ function iPipeTo(
 		return str.length > maxWidth ? str.substr(0, maxWidth) + "..." : str;
 	}
 
+	function getDefaultChoices(promptType) {
+		if (promptType.type === "list") {
+			return options.default;
+		}
+
+		if (promptType.type === "checkbox") {
+			return options.default.split(
+				options["default-separator"] || options.separator || os.EOL
```

Comment:
This falls back to `separator`, otherwise `os.EOL`, similar to [this line](https://github.com/ruyadorno/ipt/blob/master/src/cli.js#L68).